Vision/Vision.bs

```diff
@@ -123,8 +123,7 @@ Markup Shorthands: markdown yes
 
 	<ul>
 	<li id=user-first>
-		**User-first**: We put the needs of users first:
-		above authors, publishers, implementers, paying W3C Members, or theoretical purity.	
+		**User-first**: We prioritize the needs of users before other constituencies, including those of W3C Members.
```

Comment:
There's something off about the way "including those of W3C Members" attaches to the rest of the sentence. Not sure how you want to fix it, but maybe this works?

```suggestion
		**User-first**: We prioritize the needs of users over other constituencies, including over those of W3C Members.
```